Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -270,7 +270,7 @@ public void onPlayerQuit(final PlayerQuitEvent event) {
             final String msg = ess.getSettings().getCustomQuitMessage()
                 .replace("{PLAYER}", player.getDisplayName())
                 .replace("{USERNAME}", player.getName())
-                .replace("{ONLINE}", NumberFormat.getInstance().format(ess.getOnlinePlayers().size()))
+                .replace("{ONLINE}", NumberFormat.getInstance().format(ess.getOnlinePlayers().size() - 1))
```

Comment:
```suggestion
                .replace("{ONLINE}", NumberFormat.getInstance().format(ess.getOnlinePlayers().size() - 1)) // Subtract 1 as the leaving player is still online during this time
```

Reply:
Doesn't seem like there is much of an alternative here, but given, these placeholders are only being processed in the quit event and not elsewhere, so it seems reasonable enough